Add use case of Computing-Aware AI large model (#1)
cats-wg/draft-ietf-cats-usecases-requirements: draft-ietf-cats-usecases-requirements.md

   WAN1 -> Cloud2 vCPE1 -> Cloud2 APP1

##  Computing-Aware AI large model
   
   AI large model refers to a type of artificial intelligence model that 

>    is trained on massive amounts of data using deep learning techniques. 

Reviewer: Can we have a citation here?

Author: Actually, it is not a direct citation, but a rewording of several citations, including:

- https://www.merantix.com/blog/the-significance-of-ai-models#:~:text=Large%20AI%20Models%2C%20also%20known,are%20used%20to%20build%20them.
- https://aws.amazon.com/cn/blogs/aws/generative-ai-with-large-language-models-new-hands-on-course-by-deeplearning-ai-and-aws/#:~:text=Generative%20AI%20is%20powered%20by,across%20many%20natural%2Dlanguage%20tasks.
- https://techcommunity.microsoft.com/t5/ai-machine-learning-blog/training-large-models-in-azure-machine-learning/ba-p/3834865

Is it ok to list several citation links?

Reviewer: I would pick one of these and cite it.

Author: Done. Please take a look

   These models are characterized by their large size, high complexity, 
   and high computational requirements. AI large models have become 
   increasingly important in various fields, such as natural language 

>    processing, computer vision, and speech recognition.

Reviewer: I would delete the last sentence. It does not add much.

Author: Agree and deleted.

>    There are usually two types of AI large models, AI foundation model 

Reviewer: The subtle "usualy" may be confusing for some readers absent an authoritative reference to cite;

Author: Agree to delete "usually".

   and customized model. AI foundation large model is a model that can 
   handle multiple tasks and domains, and has wider applicability and 
   flexibility, but may not perform as well as customized model in 

>    specific domain tasks. Customized model is trained for specific 

Reviewer: ```suggestion
   specific domain tasks. Customized models are trained for specific 
```

   industries or domains, and more focused on solving specific problems, 

>    but may not be applicable to other domains. AI foundation model 

Reviewer: ```suggestion
   but may not be applicable to other domains. AI foundation models
```

>    usually involve mega-scale parametere, while customized model involves 

Reviewer: What is a "mega-sclae parametere"?

Author: Sorry, typo, it should be "mega-scale parameters"

Reviewer: I saw the nit, but I don't see what is a "mega-scale parameter". I see what is a massive scale DC, data, etc. but not sure what these parameters are.

Reviewer 2: From [ourworldindata](https://ourworldindata.org/grapher/artificial-intelligence-parameter-count#:~:text=Parameters%20are%20variables%20in%20an,in%20an%20artificial%20neural%20network.):
`Parameters are variables in an AI system whose values areadjusted during training to establish how input data get stransformed into the desired output.`

When training, the parameter is loaded into the GPU memory. So more parameters requires more GPUs to train it. 

Can we give a number for mega-scale parameter? For example, billion/trillion? Or how much resource it takes to train it.

Author: This [link](https://infohub.delltechnologies.com/l/generative-ai-in-the-enterprise/large-model-training/#:~:text=Large%20generative%20AI%20models%20have,single%20NVIDIA%20Tesla%20V100%20GPU.) gives an example.

Large generative AI models have significant compute requirements for training. According to OpenAI, for Chat GPT-3 with 175B parameters, the model size is approximately 350 GB, and it would take 355 years to train GPT-3 on a single NVIDIA Tesla V100 GPU.

So, is it ok to change to "AI foundation model usually involve billions of parameters"?

>    usually involve mega-scale parametere, while customized model involves 

Reviewer: ```suggestion
   usually involve mega-scale parameters, while customized models involve 
```

   large or middle-scale parameters.
   

>    Also, AI large model contains two key phases: training and inference. 

Reviewer: ```suggestion
   Also, AI large model involves two key phases: training and inference. 
```

Is this really specific to these models?

Author: Good question. Actually, training is a universal phase that each AI model need to do, while inference is not. AI large model, in most cases refers to large language model, involves inference. But for some other AI models, like a model trained to categorize images, inference is not needed.

Reviewer 2: I am not sure about that. From [Google AI model inference](https://www.google.com/search?q=ai+model+inference):
`Machine learning inference is the process of running live data into a machine learning algorithm to calculate output such as a single numerical score.`

It seems categorize images is also inference.

Maybe the difference between the large model and traditional ai model is that it can generate a lot of content(so called AIGC: AI Generated Content). But I am not sure about what the difference it makes related to CATS.

Author: By looking at different citations, I think this one from [xilinx](https://www.xilinx.com/applications/ai-inference/difference-between-deep-learning-training-and-inference.html#:~:text=Machine%20learning%20inference%20is%20the,sort%20different%20fruits%20by%20color.) is worth consideration

Machine learning inference is the ability of a system to make predictions from novel data.

So, in this case, I agree classifying images is an inference, since a machine learning model trained to classify images of cats and dogs can be used to infer whether a new image is of a cat or a dog.

And so, inference can be a common ability of AI models.

> Maybe the difference between the large model and traditional ai model is that it can generate a lot of content(so called AIGC: AI Generated Content).

In my view, why AI large model has billions of parameters is because it deals with natural languages.

> But I am not sure about what the difference it makes related to CATS.

I think if an AI large model aims to generate images or videos, then it will often bring on high demand on network resource and computing resource, which is the area related to CATS.

   WAN1 -> Cloud2 vCPE1 -> Cloud2 APP1

##  Computing-Aware AI large model

   A number of industry associations, such as the Industrial Digital
   Twin Association or the Digital Twin Consortium
   (https://www.digitaltwinconsortium.org/), have been founded to
   promote the concept of the Digital Twin (DT) for a number of use case
   areas, such as smart cities, transportation, industrial control,
   among others.  The core concept of the DT is the "administrative
   shell" {{Industry4.0}}, which serves as a digital representation of the
   information and technical functionality pertaining to the "assets"
   (such as an industrial machinery, a transportation vehicle, an object
   in a smart city or others) that is intended to be managed,

>    controlled, and actuated.

Reviewer 2: What is the relation between digial twin and AI large model?

Author: Sorry, typo. I copied the digital twin first paragraph to follow the format. Forgot to delete
